server/chat-commands/core.ts

```diff
@@ -937,21 +938,23 @@ export const commands: ChatCommands = {
 		if (target) {
 			const parsed = parseInt(target);
 			if (parsed > 6) return this.errorReply(this.tr`Use a number between 1-6 to view a specific set.`);
-			if (isNaN(parsed)) {
```

Comment:
I can't review on the if (target) line, but you should put the !hideStats check there instead, I believe. 
if (target && !hideStats) ...

Reply:
Would simplify some code.

Reply:
It could run the else statement then though, which I don't think we want, since it would try to display the info an arrayIndex of whatever "hideStats" - 1 is which it probably won't like.

Reply:
Oh never mind I'm dumb, was looking at the wrong line lololol